### ISSUE TITLE
File Set Landing Page Enhancements

### ISSUE THREAD

**Issue by ntallman** [psu-libraries/cho#796]
As a Chromebook user,
I would like to know more information (filename, file format, and size) about the files available for download,
So that I can choose a file appropriate to my need and available storage.

Enhancement: #63, #649

Blocked by #649

**Comment by ntallman** [comment]
My thought was a small table on the file set landing page. Ability to download all somehow would be nice.

**Comment by awead** [comment]
TODO:
* display listing of files as a table
* three columns: name, size, and mime type
* derive mime type from file name (for now)
* display the `friendly` name of the mime type via `MIME::Types`
* links in the name download the file

**Comment by ntallman** [comment]
@awead Is this blocked by #66 as a place to store the technical metadata? (filename, file format, and file size)  Should we report file size in decimal or binary byte form?

**Comment by rschenk** [comment]
In the column that shows the `friendly` text version of Mime Type, this friendly version is not available on all mime types. Most notably, jp2 / jpeg-2000 does not have a friendly name by default, and calling that method returns nil. We can add "friendlies" ourselves, but I don't think it's realistic that we can be exhaustive here for ever file conceivably uploaded to CHO. That means we'll need something else to fall back upon to display in this column, which I decided should be the file extension. 

Falling back on the file extension works great,  but upon seeing all the friendly mime type text in a table, along with the "jp2" extension, that made me wonder if the extension is actually more usable. If you were looking to download the raw TIFF of a file, would you rather see "tif" or "Tagged Image File Format" — personally I didn't even know that's what TIFF stood for! Just some food for thought @ntallman @awead

**Comment by ntallman** [comment]
Thanks @rschenk! I haven't actually seen the output from `friendly` yet, you make a good point that some of them may not be all that friendly compared to the extension. We might want to go ahead an add something friendly for JPEG 2000, since we know we're going to have tons of them. Can we contribute friendlies upstream or only add them locally?

We are showing the filename, which includes the extension. If they do see that mouthful for TIFF, they'll still see the .tiff in the filename. Not as easy to parse on a glance though. Not necessarily something to do right now, I'd like to see what this live in the app to get a feel, but we could show both. We could have the friendly (falling back to mime type) in one column and have the extension in another column.  When CHO starts handling born-digital content (2.x?) we'll start seeing a lot more formats.
